docs/libdnf/version.xml.in

```diff
@@ -0,0 +1 @@
+@DNF_VERSION@
```

Comment:
This needs to be changed to `@LIBDNF_VERSION@` to work correctly with CMake's substitution stuff.